content/saas/opsworks_migration.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Migrate from AWS OpsWorks to Chef SaaS"
+draft = false
+[menu]
+  [menu.saas]
+    title = "OpsWorks Migration"
+    identifier = "chef_infra/OpsWorks Migration"
+    parent = "chef_saas"
+    weight = 30
++++
+
+This guide will walk you through the migration scenarios from AWS OPSWorks to Chef SaaS.
+
+{{< note >}} Assuming that you have upgraded to Automate 2.0 {{< /note >}}
+
+If you have already upgraded to Chef Automate 2.0, no further installation, upgrade, or maintenance is required.
+
+## Prerequisites
+
+* Running Automate 2.0 in AWS OpsWorks.
+
+* A Chef SaaS Environment. Refer to the [Getting Started](/get_started/) with the Chef SaaS page.
+
+* An S3 bucket was provided by Progress Chef.
```

Comment:
A few things:

- every list needs an introductory sentence explaining the list
- list items shouldn't have empty lines between them
- verify the URL 
- don't use the past tense